### DISCUSSION TITLE
Experiencing Slowness on GitHub?

### DISCUSSION THREAD

**Discussion by diegocatao** [community/community#76354] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hey all,

Lately, I've been noticing that pages on GitHub are taking longer to load than usual. This seems to be affecting various parts of the platform. I wanted to check if anyone else is experiencing similar issues.

I'm trying to gather some insights and determine if this is a widespread issue or if there's something specific to my local network.

**Comment by arimartorano** [comment]
Hey! So, no issues from me at all. My GitHub is running as usual. Maybe check the speed of your computer or see how many people are using the wifi at once.
